src/scripts/constants.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+class _constants:
+
+    _DEFAULT_TEST_COUNT = 100 # The default value of --test-count option
```

Comment:
If you use hidden classes to group constants, you can ignore the '_' prefix in case of member properties.
These are looks ugly:
```
DEFAULT_TEST_COUNT
constants.DEFAULT_TEST_COUNT
constants._DEFAULT_TEST_COUNT
_constants.DEFAULT_TEST_COUNT
_constants._DEFAULT_TEST_COUNT
```
It is much better to see something like:
```
_DEFAULT_TEST_COUNT
```
or even better:
```
_constants.default_test_count
```

So, if you are about to use variables for predefined values, please use the following schema:
```
_<group_name_of_values>.<key>
```